samples/authorization/csm_authorization_proxy_server_v230.yaml

```diff
@@ -81,6 +81,10 @@ spec:
               redisUsernameKey: ""
               # Key in the secret that stores the Redis password.
               redisPasswordKey: ""
+              # Paths for the Conjur secrets that store the Redis username and password
```

Comment:
I guess this file too needs to be updated.
config/samples/storage_v1_csm_authorization_v2.yaml